pkg/review/config.go

```diff
@@ -30,7 +30,6 @@ import (
 type Config struct {
 	GitHubEnterpriseServerURL string `env:"GITHUB_ENTERPRISE_SERVER_URL"`       // The GitHub Enterprise Server instance URL, format "https://[hostname]"
 	GitHubAppID               string `env:"GITHUB_APP_ID,required"`             // The GitHub App ID
-	GitHubInstallID           string `env:"GITHUB_INSTALL_ID,required"`         // The provisioned GitHub App Installation reference
```

Comment:
@bradegler ended up adding the change as it turned out not much refactor needed, please take another look, though it does not include updating the terraform modules etc.

Reply:
This looks good. Are you planning on running the artifacts job too at some point? That will need a similar change.

Reply:
Not for GHSS, but I can take a look at it now.